src/main/java/uk/ac/cam/cl/dtg/segue/dao/users/PgUsers.java

```diff
@@ -47,6 +47,7 @@
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 import org.apache.commons.lang3.Validate;
+import org.jetbrains.annotations.Nullable;
```

Comment:
The one from `jakarta` seems to be the one being used across the whole codebase.

Reply:
Oh, not again. That seems to sometimes automatically added when extracting or copying functions. Must have failed to notice this time...
Given the other findOne methods I used as a basis for that function don't use it, I'm pretty sure it can just be removed there as well...